README.md

```diff
@@ -0,0 +1,69 @@
+# Public ACRL Sites
+
+`www` is the repo for the following public ACRL sites 
+- [ACRL](http://appliedcomputing.io) - home of Applied Computing Research Labs
+- [SimKube](http://simkube.dev/) - home of SimKube
+
+Both sites are:
+- built into static assets using [mkdocs](https://www.mkdocs.org/) for static site generation
+- automatically re-deployed via a GitHub action on successful merges to the `master` branch
+
+
+## Getting Started
+
+### Prerequisites
+
+- Python 3 (^3.11)
+- Poetry (for dependency management)
+- Git (for version control)
+- Just (command runner)
+
+
+## Development
+
+### Installation
+
+1. `cd` to your preferred directory
+2. Clone the repository:
+   ```sh
+   git clone https://github.com/acrlabs/www.git
+   cd www
+   ```
+
+### Running Site Previews
+
+1. `cd` to the base directory of your local copy `www` repo
+2. Initialize the poetry virtual environment and install dependencies with:
+   ```sh
+   poetry install
+   ```
+
+3. Run the `serve` command:
+    ```sh
+   just serve {site}
+   ```
+
+   Replace `{site}` with either `acrl` (for ACRL site) or `simkube` (for SimKube site), example:
+    ```sh
+   just serve acrl
+   ```
+
+4. The site will now be accessible at http://localhost:8000
```

Comment:
minor note, the simkube site launches at :8001 so that you can have both sites active at the same time.